src/content/extensions/multipleroutes/src/README.md

```diff
@@ -1,15 +1,18 @@
 ## Chrome WebRTC Network Limiter
-Disables the WebRTC multiple-routes option in Chrome's privacy settings.
+Configures the WebRTC traffic routing options in Chrome's privacy settings.
 
 ★ What it does:
-This configures WebRTC to not use certain IP addresses:
+This configures WebRTC to not use certain IP addresses or protocol:
 - IP addresses not visible to the public internet (e.g. addresses like 192.168.1.2)
 - any public IP addresses associated with network interfaces that are not used for web traffic (e.g. an ISP-provided address, when browsing through a VPN)
+- Require WebRTC traffic to go through proxy servers as configured in Chrome. Since most of the proxy servers don't handle UDP, this effectively turns off UDP for now, until UDP proxy support is available in Chrome and such proxies are widely deployed.
  
 Once the extension is installed, WebRTC will only use public IP addresses associated with the interface used for web traffic, typically the same addresses that are already provided to sites in browser HTTP requests.
 
+However, this will not disable non-proxied UDP by default. It could be configured through the extension's Options page.
```

Comment:
Perhaps something like "The extension may also disable non-proxied UDP, but this is not on by default and must be configured using the extension's Options page."